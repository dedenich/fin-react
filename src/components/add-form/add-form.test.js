import React from 'react';
import renderer from 'react-test-renderer';
import AddForm from "./add-form.jsx";


it(`withForm is rendered correctly`, () => {
  const tree = renderer
  .create(
      <AddForm
        isAimAdded={false}
        onCloseButtonClick={jest.fn()}
        onInputChange={jest.fn()}
        onFormSubmit={jest.fn()}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
