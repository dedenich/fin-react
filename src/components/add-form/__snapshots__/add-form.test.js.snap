// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withForm is rendered correctly 1`] = `
<div
  className="form-bg"
  id="form"
>
  <div
    className="wrapper"
  >
    <button
      className="close_button"
      onClick={[MockFunction]}
    />
    <div
      className="card form"
    >
      <h3>
        Добавление цели
      </h3>
      <form
        action="#"
        onSubmit={[MockFunction]}
      >
        <input
          autoFocus={true}
          name="amount"
          onChange={[MockFunction]}
          placeholder="Сумма"
          step="0.01"
          type="number"
        />
        <input
          name="comment"
          onChange={[MockFunction]}
          placeholder="Комментарий"
          type="text"
        />
        <button
          className="form_submit"
          type="submit"
        >
          Добавить!
        </button>
      </form>
    </div>
  </div>
</div>
`;
